core/comet/service.go

```diff
@@ -0,0 +1,63 @@
+package comet
+
+import (
+	"context"
+	"time"
+)
+
+// InfoService is an interface that can be used to get information specific to Comet
+type InfoService interface {
+	GetCometInfo(context.Context) BlockInfo
+}
```

Comment:
How about we do a `Misbehaviors` wrapper like we did for `VoteInfos` instead of an array

Reply:
modified thanks